src/PowerShellEditorServices/Console/ConsoleReadLine.cs

```diff
@@ -19,8 +19,9 @@ namespace Microsoft.PowerShell.EditorServices.Console
 
     internal class ConsoleReadLine
     {
-        #region Private Fields
+        #region Private Field
 
+        private bool isUnix = false;
```

Comment:
C# fields are automatically initialized to their default values which is false for a bool.